sound/soc/codecs/rt1308-sdw.h

```diff
@@ -140,6 +140,7 @@ static const struct reg_default rt1308_reg_defaults[] = {
 	{ 0x3008, 0x02 },
 	{ 0x300a, 0x00 },
 	{ 0xc003 | (RT1308_DAC_SET << 4), 0x00 },
+	{ 0xc000 | (RT1308_POWER << 4), 0x00 },
 	{ 0xc001 | (RT1308_POWER << 4), 0x00 },
 	{ 0xc002 | (RT1308_POWER << 4), 0x00 },
```

Comment:
how about change sequence from 0xc000 to 0xc003 ?

Reply:
The order is from smallest to biggest. The RT1308_DAC_SET is 0x10, however RT1308_POWER is 0x32.